declarations/Facebook.json

```diff
@@ -15,12 +15,18 @@
       "executeClientScripts": true
     },
     "Privacy Policy": {
-      "fetch": "https://www.facebook.com/about/privacy",
+      "fetch": "https://mbasic.facebook.com/privacy/policy/printable/",
       "filter": [
-        "removeReturnToTopButtons",
-        "cleanUrls"
+        "removeEmptyAnchorsLinks",
+        "removeTrackingIDs",
+        "removeLocaleFromUrls"
```

Comment:
`locale2` query param does not seem to me to be present in the urls of the document, so the filter should not be necessary